Use forward declaration for NETWinInfo class
instead of including KDE/NETWinInfo header,
which in turn includes much X stuff
--- a/libs/taskmanager/task.h
+++ b/libs/taskmanager/task.h
@@ -34,11 +34,12 @@ CONNECTION WITH THE SOFTWARE OR THE USE 
 // KDE
 #include <KDE/KSharedPtr>
 #include <KDE/KWindowSystem>
-#include <KDE/NETWinInfo>
 
 #include <taskmanager/taskmanager.h>
 #include <taskmanager/taskmanager_export.h>
 
+class NETWinInfo;
+
 namespace TaskManager
 {
 
--- a/libs/taskmanager/task.cpp
+++ b/libs/taskmanager/task.cpp
@@ -40,6 +40,7 @@ CONNECTION WITH THE SOFTWARE OR THE USE 
 #include <KDebug>
 #include <KIconLoader>
 #include <KLocale>
+#include <NETWinInfo>
 
 #include "taskmanager.h"
 
--- a/libs/taskmanager/taskmanager.cpp
+++ b/libs/taskmanager/taskmanager.cpp
@@ -35,6 +35,7 @@ CONNECTION WITH THE SOFTWARE OR THE USE 
 #include <KDebug>
 #include <KGlobal>
 #include <KLocale>
+#include <NETWinInfo>
 
 #ifdef Q_WS_X11
 #include <QX11Info>
